.travis.yml

```diff
@@ -1,49 +1,27 @@
-env:
-  global:
-    - RUST_BACKTRACE=1
-    - PATH=$PATH:$HOME/.cargo/bin:$HOME/rust/bin
-
 sudo: true
 dist: trusty
-
 os:
-  - linux
-  - osx
-osx_image: xcode8.2
-
-compiler:
-  - gcc
-  - clang
-
+- linux
 language: node_js
-
 node_js:
-  - "7"
-
-addons:
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
-    packages:
-      - g++-4.8
-      - desktop-file-utils
-      - xdg-utils
-
-cache:
-  cargo: true
-  directories:
-    - node_modules
-
-before_install:
-  - "[[ ${TRAVIS_OS_NAME} = linux ]] && export CXX=g++-4.8 || echo 'skipped'"
-
+- '7'
 install:
-  - node --version
-  - npm --version
-  # also downloads the build for release version
-  - NODE_ENV=dev npm install
-
+- node --version
+- npm --version
+- NODE_ENV=dev npm install
 script:
-  - npm run lint
-  - npm run test-coverage
-  - npm run publish-coverage
+- npm run lint
+- npm run test-coverage
+- npm run publish-coverage
+before_deploy:
+- rm -rf node_modules
+- unset NODE_ENV
+- npm install
```

Comment:
for consistency let's see if we can use `yarn` for this. Not sure if that's possible from the CI point of view (do they have `yarn` as a provider?). But worth a look. If not, then `npm` is fine for our purposes here.

Reply:
We don't have to run npm install or yarn install for `safe-app`. It makes sense to install dev dependencies for `beaker-plugins-* ` since we have to use babel. The node_modules are ignored during publish. However in the script section we should be using `yarn` instead of npm. 

@joshuef, yarn is provided by [default](https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn)